app/api/utils.py

```diff
@@ -451,6 +471,67 @@ def paint(self, documents):
         return res
 
 
+class SpacyPainter(JSONPainter):
+
+    def paint(self, documents):
+        data = super().paint(documents)
+        res = []
+
+        label_ids = {a['label'] for d in data for a in d['annotations']}
+        label_texts = {l['id']: l['text'] for l in Label.objects.filter(id__in=label_ids).values('text', 'id')}
```

Comment:
We could avoid this database lookup if in the annotation serializers ([DocumentAnnotationSerializer](https://github.com/doccano/doccano/blob/72cafaf881aa239393566278613a725c35b60ee8/app/api/serializers.py#L162) and [SequenceAnnotationSerializer](https://github.com/doccano/doccano/blob/72cafaf881aa239393566278613a725c35b60ee8/app/api/serializers.py#L173)) we serialized the label field as an object (e.g. `{"id": 1, "text": "the-label"}`) instead of just an id. This would also be beneficial for the CSV/JSONL data export where currently we're exporting labels as ids instead of as human readable text. @Hironsan what are your thoughts about changing the annotation serializers to include a text version of the label instead of only the id?